src/pages/PromptPage/PromptPage.jsx

```diff
@@ -0,0 +1,106 @@
+import "./PromptPage.scss";
+import { useState } from "react";
+import Button from "../../components/Button/Button";
+import InputBox from "../../components/InputBox/InputBox";
+import PreviewBox from "../../components/PreviewBox/PreviewBox";
+import { getResponseContent } from "../../utils/openAIcall";
+
+const requestObj = {
+  messages: [
+    {
+      role: "system",
+      content:
+        "You are a friendly assistant, that gives responses to a community organizer appropriate to LinkedIn in JSON format just post content between 200 and 2000 characters in length, no title. The key should be 'content'. Don't include any extra text outside of the post content itself, including hashtags. Don't say you will create the post, just give me the content",
+    },
+    {
+      role: "user",
+      content: "",
+    },
+  ],
+  model: "gpt-3.5-turbo",
+  temperature: 0.02,
+};
+
+const PromptPage = () => {
+  const [inputText, setInputText] = useState("");
+  const [previewText, setPreviewText] = useState("Preview here");
+  const [isEditing, setIsEditing] = useState(false);
+  const [aiResponseLoading, setAiResponseLoading] = useState("");
```

Comment:
I'm getting a warning about aiResponseLoading never being used (it starts as empty string, then gets set to true, and I don't think is used itself). Can we just use the previewText (or something similar)?